component/src/test/java/org/wso2/extension/siddhi/map/text/sourcemapper/TextCustomSourceMapperTestCase.java

```diff
@@ -639,4 +647,64 @@ public void receive(Event[] events) {
         assertEquals(count.get(), 2);
         siddhiAppRuntime.shutdown();
     }
+
+    /**
+     * Creating test for publishing events with Text mapping.
+     *
+     * @throws Exception Interrupted exception
+     */
+    @Test
+    public void testTextMappingSingleCustom() throws Exception {
+        AtomicInteger eventCount = new AtomicInteger(0);
+        int waitTime = 50;
+        int timeout = 30000;
```

Comment:
You have already defined class variables for these two: waitTime and timeout; so you can use them without redefining here.

Reply:
Fixed in cdb7b86019f370bcb90c545043b2ad205e8e98df